overview.md

```diff
@@ -11,13 +11,15 @@ However, as this idea evolved, we ran into a big problem: we don't know who
 deserves credit in an open-source project, and thus we don't know how to
 distribute the rewards. The obvious approaches, like counting commits or lines
 of code, would be far too easy to game—and would in any case miss out on vital
-work like design or issue triaging.
+work like design or issue triaging, not to mention that it would miss out on the
+underlying quality of the commits or code.
 
 This inability to quantify contributions accurately affects every method for
 funding open-source. Projects need a fair system for deciding how to distribute
 rewards regardless of whether they come through sponsorships, donations,
 [cryptocurrency airdrops], or through any [other means]. SourceCred is designed
-to do just that.
```

Comment:
I like the punchiness of "SourceCred is designed to do just that". Happy to have the additional concept in a separate sentence, e.g.:
"SourceCred is designed to do just that. We hope that these quantitative measurements of value will enable downstream work to fund open-source contributors".
(although I think this proposed sentence is also a little vague)